ansible.cfg

```diff
@@ -0,0 +1,24 @@
+[defaults]
+forks = 50
+
+# SSH timeout
+timeout = 30
+host_key_checking = False
+
+inventory = ./inventory
```

Comment:
This is usually used with infrared, right? Should it not be `infrared workspace inventory` that is the suggested inventory? 

Is this necessary?
If this is added, a note in the README is necessary to specify that there's a default inventory file, currently, users are told to provide their own and what host groups are required

Reply:
Added note to the readme.